niri-config/src/lib.rs

```diff
@@ -1031,6 +1031,8 @@ pub struct Struts {
 pub struct HotkeyOverlay {
     #[knuffel(child)]
     pub skip_at_startup: bool,
+    #[knuffel(child)]
+    pub hide_unbound: bool,
```

Comment:
Maybe `hide_not_bound` so it matches the "(not bound)" string? Or is that too weird

Reply:
i don't mind that!